### DISCUSSION TITLE
Nitpick about naming question

### DISCUSSION THREAD

**Discussion by nyteshade** [compnerd/swift-win32#750] [General]
Should we be using the UI prefix as done on iOS or NS prefix, or something entirely different, than NS for Win32 APIs and components?

**Comment by compnerd** [comment]
`NS` and `UI` should be treated as reserved prefixes as they would collide with potential handling for their common use (`NS` being reserved as NeXTStep's namespace, `UI` being reserved as UIKit's namespace).  I would consider a terse prefix if there is a good prefix.

`NS` is additionally a poor choice IMO as it would further blur and complicate the boundary between Foundation and non-Foundation APIs.
